### PR TITLE
Allowed user input to change url for get request

### LINKED ISSUE #6: Textbox component
**Use Flow:**

- [x] Enter the information (search component)
- [x] Hit search 
- [x] Send request to backend (http request module)
- [x] Get response back (pass the response to map viewer component)

**Keep in mind**: 
- [Figma](https://www.figma.com/file/DrBN5PgOzKly22MbrSi2CZ/LAR---frontend?type=design&node-id=0-1&t=9Wib0Wu6ZekE1u1Z-0)

See repo source code for updates

### PR DESCRIPTION
**Pull request by PruthviSamirPatel**
User can input the city location, which will then change the location for the request to backend upon clicking the submit button

### CODE REVIEW

File: src/Components/SearchBox.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+      <input
+        type="text"
+        onChange={props.handleChange}
+        value={props.searchText}
+        name="cityInput"
+        placeholder="Lafayette, IN"
+        title="Enter the location of interest"
+      />
+      <button type="button" onClick={props.handleSubmit}>🔍</button>
     </div>
```
Comment (wzjoriv) [inline src/Components/SearchBox.js]: When I press enter, it reloads the page. Can you make it so it doesn't but instead it executes handleSubmit?

Comment (wzjoriv) [reply src/Components/SearchBox.js]: @PruthviSamirPatel

Comment (PruthviSamirPatel) [reply src/Components/SearchBox.js]: Yes, I have fixed it in my latest commit.

**Review by wzjoriv — APPROVED** [review]

**Comment by wzjoriv** [comment]
Nice Job 👍

**Pull request merged**